model/host/host.go

```diff
@@ -1987,7 +1987,7 @@ func GetContainersOnParents(d distro.Distro) ([]ContainersOnParents, error) {
 func getNumNewParentsAndHostsToSpawn(pool *evergreen.ContainerPool, newHostsNeeded int, ignoreMaxHosts bool) (int, int, error) {
 	existingParents, err := findUphostParentsByContainerPool(pool.Id)
 	if err != nil {
-		return 0, 0, errors.Wrap(err, "could not find running parents")
+		return 0, 0, errors.Wrap(err, "could not find up and eventually running parents")
```

Comment:
I think this error message is a little strange, partially because "up" already implies "eventually running and already running). Maybe you can just say parents, or uphost parents is fine really, since these errors are for us and we know what that means.